.github/workflows/changelog-updater.yml

```
@@ -9,19 +9,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
         with:
-          ref: main
-
+          repository: helmholtz-analytics/heat
+          ref: ${{ github.event.release.target_commitish }}
       - name: Update Changelog
-        uses: stefanzweifel/changelog-updater-action@v1
-        with:
-          release-notes: ${{ github.event.release.body }}
-          latest-version: ${{ github.event.release.name }}
-
+        run: |
+          echo ${{ format('\# {0} - {1}', env.GITHUB_REF, github.event.release.name) }} > cl_title.md
+          echo ${{ github.event.release.body }} > cl_new_body.md
```

```suggestion
        env:
          NAME: ${{ github.event.release.name }}
          BODY: ${{ github.event.release.body }}
        run: |
          echo ${{ format('\# {0} - {1}', env.GITHUB_REF, "$NAME") }} > cl_title.md
          echo "$BODY" > cl_new_body.md
```
These lines may have a security risk
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#understanding-the-risk-of-script-injections